File: visitantes/src/app/models/trabajador.ts
import {CondominoInfo} from "./condominoInfo";
import {WorkDay} from "./WorkDay";

export interface Trabajador {
  id: string;
  type: string;
  name: string;
  condominoInfo: CondominoInfo;
  schedule: string;
  telephoneNumber: string;
  workDays: WorkDay[];
}

File: Downloads/del-parque/backend/src/main/java/com/example/delparque/dto/Trabajador.java
package com.example.delparque.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Builder
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Trabajador {
    private String id;
    private String type;
    private String name;
    private CondominoInfo condominoInfo;
    private String schedule;
    private String telephoneNumber;
    private List<String> workDays;
}